### ISSUE TITLE
CPU optimization

### ISSUE THREAD

**Issue by donald-mason23** [nanos-world/nanos-world-sandbox#26]
I noticed today if there are a lot of objects on the map and playing single player, the load on the CPU about 60-67%. Need to do CPU little optimization.

**Comment by Nogitsu** [comment]
If I remember well it's an issue due to entities being made on the Lua side and it's been in constant enhancements

**Comment by gtnardy** [comment]
yeah CPU usage escalates with the amount of objects spawned, we are constantly improving that
